ktor-client/ktor-client-features/ktor-client-tracing-stetho/src/androidMain/kotlin/io/ktor/client/features/tracing/KtorInterceptorRequest.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.features.tracing
+
+import com.facebook.stetho.inspector.network.*
+import io.ktor.client.request.*
+import io.ktor.http.content.*
+
+/**
+ * Implementation of [NetworkEventReporter.InspectorRequest] that is built to work with [StethoTracer].
+ */
+internal class KtorInterceptorRequest(private val requestId: String, private val requestData: HttpRequestData) :
```

Comment:
codestyle

Reply:
If you are talking about the line length it's fine.

Reply:
I'm about ':'

Reply:
Well, I still don't understand the problem.